lib/network/modules/components/nodes/util/NodeBase.js

```diff
@@ -43,6 +55,13 @@ class NodeBase {
     labelModule.adjustSizes(this.margin)
   }
 
+  /**
+   *
+   * @param {CanvasRenderingContext2D} ctx
+   * @param {Number} angle
+   * @returns {Number}
+   * @private
```

Comment:
OK, this is a gray area. My interpretation is that underscored methods are *private to the module*. Thus, it's internal to `vis.js`, not part of the API and should therefore not be used outside of it. Makes sense?

I got this definition from another project, can't recall which. But it's a workable thing.